src/pages/Order/components/CartList.js

```diff
@@ -63,11 +69,14 @@ export default class CartList extends Component {
 
   calculateTotalPriceInCart = () => {
     const { productInCart } = this.state;
-    return productInCart.reduce((acc, cur) => acc + cur.total_price, 0);
+    const checkedProduct = [];
+    productInCart.forEach(item => item.is_checked && checkedProduct.push(item));

```

Comment:
이 경우도 결국 새로운 배열을 리턴하는 결과이기 때문에 map을 사용해주면 어떨까요?

Reply:
조건에 맞는 요소만 새 배열에 담기도록 filter를 사용하여 변경했는데 어떨까요?

Reply:
- 제가 확인하기로는 filter로 변경 하셨다는 코드가 아직 올라오지 않은 것 같습니다. 확인해주세요.
- 배열 => 숫자로 나가야 하기 때문에 1) filter, 2) reduce 메서드를 체이닝하여 로직을 구성할 수 있을 것 같습니다.
```suggestion
    const totalPrice = productInCart
        .filter(item => item.is_checked);
        .reduce((acc, ({ total_price })) => acc + total_price, 0)
```

Reply:
간단하게 줄일 수 있던 코드였네요! 감사합니다! 수정하였습니다.